OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -10,260 +10,306 @@
-            }
-            var toggle = GetToggleInput(key);
-            if (toggle != null)
-            {
-                return toggle.Value;
-            }
-            var textInput = GetTextInput(key);
-            if (textInput != null)
-            {
-                return textInput.Value;
-            }
-            var numberInput = GetNumberInput(key);
-            if (numberInput != null)
-            {
-                return numberInput.Value;
-            }
-            _console.WriteLine("Error: no input found with name " + key);
-            return null;
-        }
-
-        public void SetInputValue(string key, object value)
-        {
-            var slider = GetSliderInput(key);
-            if (slider != null)
-            {
-                var val = value is JObject obj ? obj["value"] : value;
-                slider.Value = Convert.ToSingle(val);
-                return;
-            }
-            var toggle = GetToggleInput(key);
-            if (toggle != null)
-            {
-                var val = value is JObject obj ? obj["value"] : value;
-                toggle.Value = Convert.ToBoolean(val);
-                return;
-            }
-            var textInput = GetTextInput(key);
-            if (textInput != null)
-            {
-                var val = value is JObject obj ? obj["value"] : value;
-                textInput.Value = Convert.ToString(val);
-                return;
-            }
-            var numberInput = GetNumberInput(key);
-            if (numberInput != null)
-            {
-                var val = value is JObject obj ? obj["value"] : value;
-                numberInput.Value = Convert.ToSingle(val);
-                return;
-            }
-            _console.WriteLine("Error: no input found with name " + key);
-        }
-
-        protected void InvokeOnInit()
-        {
-            OnInit?.Invoke();
-        }
-
-        public void SelectFirst()
-        {
-            var firstSelectable = Menu.GetComponentInChildren<Selectable>();
-            Locator.GetMenuInputModule().SelectOnNextUpdate(firstSelectable);
-            Menu.SetSelectOnActivate(firstSelectable);
-        }
-
-        public void UpdateNavigation()
-        {
-            var selectables = Menu.GetComponentsInChildren<TooltipSelectable>()
-                .Select(x => x.GetComponent<Selectable>())
-                .Where(x => x != null).ToList();
-            for (var i = 0; i < selectables.Count; i++)
-            {
-                var upIndex = (i - 1 + selectables.Count) % selectables.Count;
-                var downIndex = (i + 1) % selectables.Count;
-                var navigation = selectables[i].navigation;
-                navigation.selectOnUp = selectables[upIndex];
-                navigation.selectOnDown = selectables[downIndex];
-                selectables[i].navigation = navigation;
-            }
-        }
-
-    }
+	public class ModMenu : IModMenu
```

Comment:
According to GitHub, everything in this file has changed. But it seems just the indents have changed. Probably it was unintended?

Reply:
done